tools/autograd/templates/Functions.cpp

```diff
@@ -2139,6 +2139,11 @@ Tensor index_backward(const Tensor & self, TensorList indices, const Tensor& gra
    return at::_index_put_impl_(zeros, indices, grad, true, true);
 }
 
+const char* ERR_BACKWARD_TWICE =
```

Comment:
Do we still need the logic in torch/csrc/autograd/saved_variable.cpp ?

Reply:
I changed it to the assertion